ui/installer/VCSA/configs

```diff
@@ -10,3 +10,6 @@ VIC_UI_HOST_URL="NOURL"
 
 # If VIC_UI_HOST_URL starts with "https", enter SHA-1 thumbprint of the web server hosting the vic-ui.zip file
 VIC_UI_HOST_THUMBPRINT=""
```

Comment:
side note if it's not documented anywhere else, what format does this expect as i've seen fingerprints with spaces, no spaces, or `:`